sass/navigation.slidein.scss

```diff
@@ -0,0 +1,229 @@
+// VARIABLES
+$transition-speed: .125s;
+$transition-easing: ease;
+$hide-desktop-trigger-below: 979px;
+$drawer--width: 20rem;
+$drawer--padding: 18px;
+$drawer--z-index: 10;
+$mask--z-index: 2;
+$pushed-content--z-index: 1;
+
+$mask--background: rgba(255, 255, 255, 0.5);
+$drawer--background-color: #686868;
+$drawer--background-color--hover: darken($drawer--background-color, 10%);
+$drawer--background-color--focus: darken($drawer--background-color, 15%);
+$drawer--border-color: #414141;
+$drawer--inner--color: #fff;
+$drawer--inner--font-size: 16px;
+
+#button--slide-in {
+  float:right;
+  color:#000;
+  border:none;
+  background-color:rgba(0, 0, 0, 0);
+  font-family: Arial,Sans-Serif;
+  padding:10px;
+  font-size:14px;
+  line-height: 1.75em!important;
+  outline: 0;
+  &:hover {
+    background-color: #002145!important;
+    color: #fff!important;
+  }
+}
+
+@import 'components/button-clf-hamburger';
+
+#pushed-content {
+  position: relative;
+  z-index: $pushed-content--z-index;
+  display: block;
+  overflow: hidden;
+  //transform: translate3D(0, 0, 0);
+  transition: transform $transition-speed $transition-easing;
+}
+#pushed-content.drawer--push-left.off-canvas-drawer--is-active {
+  transform: translate3D(-$drawer--width, 0, 0);
+  box-shadow: 2px 0 2px rgba(0, 0, 0, .15);
```

Comment:
I think this isn't rendering (as it's behind the content). We might want to remove this property.